website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -163,6 +163,35 @@ If no filters have been applied, use a [Tooltip](/components/tooltip) coupled wi
 
 ![Empty state in applied filters](/assets/patterns/filter-patterns/applied-filters-empty-state.png =450x*)
 
+### Avoiding empty state
+
+Because filtering is often reductive in nature and driven by a user action, it might not always be possible to avoid an empty state within a data set. This can be helped by leveraging specific component variants and properties, and by taking steps to prevent mistakes or errors that would result in an empty state prior to them occurring.
+
+#### ListItem count
+
+Use the `count` property within the Dropdown ListItem to more clearly explicitly call attention to filters that will return zero results.
+
+In this example, `status = pending` does not correspond with any results.
+
+![Count property in ListItem](/assets/patterns/filter-patterns/avoid-empty-state-list-item-count.png)
+
+#### Approximate matching
+
+To help resolve basic spelling mistakes, typos, and broaden the relevancy of a search term being filtered upon, consider using _approximate_ matching or "fuzzy search" when filtering by a string value.
+
+This can help reduce the burden on the user to accurately enter long and complex strings that may consist of seemingly random sequences of characters which are common characteristics of string values within a data set.
+
+!!! Info
+
+More details around the specifics of fuzzy search are beyond the scope of these guidelines, but are included to highlight a creative and user-centric search experience when filtering.
+!!!
```

Comment:
[suggestion] I worry that "beyond the scope of these guidelines" will result in people asking for us to include it. Maybe instead, this could be reframed as something like: 

We don't offer or support fuzzy search, but we've included this concept to highlight a creative and user-centric search solution when filtering that may help the user avoid an empty state.